app/views/devices_guidance/how_to_order_for_independent_special_schools.html.erb

```diff
@@ -0,0 +1,210 @@
+    </h3>
+    <p class="govuk-body">
+      We need some information from you to set up your Get help with technology service account, even if you’ve used
+      the service before. You’ll place your orders using this account.
+    </p>
+    <p class="govuk-body">
+      If you’ve been nominated to order for your LA, you’ll receive an email with a link to an online form.
+      You need to use that form to tell us:
+    </p>
+    <ol class="govuk-list govuk-list--number">
+      <li>
+        Your full name.
+      </li>
+      <li>
+        Your local authority name and 3-digit code.
+      </li>
+      <li>
+        The delivery address we should send your laptops to (all devices will be delivered to the same location at your local authority).
+      </li>
+      <li>
+        The person we should contact when we deliver your laptops.
+      </li>
+    </ol>
+    <p class="govuk-body">
+      If you’ve been nominated to order but have not received an email with a link to the form, <%= govuk_link_to 'contact us', support_ticket_path %>.
+    </p>
+    <p class="govuk-body">
+      Once you’ve completed and submitted your form, it will take up to 5 working days for us to set up your account.
+      We’ll let you know as soon as you can sign in and order.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      4. Order laptops and request internet access
+    </h3>
+    <p class="govuk-body">
+      Once you complete steps 1 to 3 and your ordering account is set up, you’ll receive an email to let you know.
+    </p>
+    <p class="govuk-body">
+      You can then sign in to the Get help with technology service to order laptops.
+      You can also request internet access for eligible pupils.
+    </p>
+    <p class="govuk-body">
+      You can order all of your allocation at once, or place multiple orders.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      5. Receive laptops
+    </h3>
+    <p class="govuk-body">
+      All of your laptops will be delivered to the local authority address you provided in your contact details form.
+    </p>
+
+    <p class="govuk-body">
+      Most deliveries will arrive within 5 working days after you place your order.
+    </p>
+
+    <h3 class="govuk-heading-m" id="step-6">
+      6. Laptop ownership, safeguarding and providing IT support
+    </h3>
+    <p class="govuk-body">
+      Your local authority will own these laptops, and be responsible for making sure they have <%= govuk_link_to 'appropriate safety
+      and security measures installed and maintained', 'devices/safeguarding-for-device-users' %>.
+    </p>
+    <p class="govuk-body">
+      You’ll be responsible for lending these laptops to state-funded pupils who need them, and redistributing them if
+      pupils move school or leave education.
+    </p>
+    <p class="govuk-body">
+      <%= govuk_link_to 'Find out about options for gifting these laptops to families and schools','/devices/device-distribution-and-ownership#who-owns-the-devices' %>.
```

Comment:
Change to: Find out about options for lending or giving devices to children, young people and other organisations.

URL is: https://get-help-with-tech.education.gov.uk/devices/device-distribution-and-ownership#who-owns-the-devices